exercises/00-init/src/modules/root/components/root.js

```diff
@@ -2,10 +2,10 @@ import React from 'react';
 import Header from './header.js'
 import UsersList from './users-list'
 
-const Root = () => (
+const Root = (props) => (
   <div>
-    <Header title="User Management" />
-    <UsersList/>
+    <Header title={props.title} />
+    <UsersList users={props.users} addUser={props.addUser}/>
   </div>
 );
```

Comment:
Missing propTypes for Root component. No worries though, eslint would let you know if it was configured on this project :) Also, you basically never get any props on `<Root>` element, so :man_shrugging:

Reply:
Nothing that can't be solved by a little copypasting ;).